src/nrniv/backtrace_utils.cpp

```diff
@@ -0,0 +1,57 @@
+#include <iostream>
+#include <cxxabi.h>
```

Comment:
Is `cxxabi.h` portable? Seems like more libstdc++ specific. (e.g. doesn't have with [MSVC](https://stackoverflow.com/questions/55650549/cannot-open-include-file-cxxabi-h-no-such-file-or-directory))
Could you take a look?

Reply:
You are right, checking for the file existance and disable demangling if not available.